packages/terra-clinical-detail-view/src/DetailList.jsx

```diff
@@ -21,27 +21,54 @@ const propTypes = {
     PropTypes.objectOf(DetailListItem),
     PropTypes.arrayOf(PropTypes.objectOf(DetailListItem)),
   ]),
+
+  /**
+   * The list type based on whether the list items are single HTML elements or label-value pairs.
```

Comment:
Should we add this UX Accessibility badge here and in the DetailListItem prop to use the appropriate list as well as the appropriate element like describe here for better accessibility?

![IMPORTANT](https://badgen.net/badge/UX/Accessibility/blue)

Reply:
Updated here: 15919a28584615751e8f0a272250c052918de4ff